.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+## What
+
+[Link to story](https://dsdmoj.atlassian.net/browse/LCAM-XXX)
+
+Describe what you did and why.
+
+## Checklist
+
+Before you ask people to review this PR:
+
+- [ ] Tests should be passing: `./gradlew test`
+- [ ] Github should not be reporting conflicts; you should have recently run `git rebase main`.
+- [ ] Avoid mixing whitespace changes with code changes in the same commit. These make diffs harder to read and conflicts more likely.
+- [ ] You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
+- [ ] You should have checked that the commit messages say why the change was made.
```

Comment:
Should we include some mention of conventional commits and that messages should try and follow this or is it a bit early yet?

Reply:
I think it's a bit early, we can update the documentation once we've agreed on the implementation :)

Reply:
I did have a section in here under the checklist which went into detail about what constitutes an approvable PR, but I've pulled it for now until we're ready to actually add it.